content/docs/user-guide/merge-conflicts.md

```diff
@@ -50,14 +50,16 @@ In `.dvc` files generated by `dvc add`, you'll get something that looks like:
 outs:
 < < < < < < < HEAD
 - md5: a304afb96060aad90176268345e10355
+  size: 12
 = = = = = = =
 - md5: 35dd1fda9cfb4b645ae431f4621fa324
+  size: 100
 > > > > > > > branch
   path: data.xml
 ```
 
-If you decide to just pick one of the versions, leave that md5 and delete the
-other one:
+If you decide to just pick one of the versions, leave that md5 (with or without
+`size` and, possibly, `nfiles` fields) and delete the other one(s):
```

Comment:
Sorry for the delay here. Just a follow-up question @efiop: What's the purpose of this parenthesis? I.e. if it's "with or without", why even mention it? Thanks

Reply:
@jorgeorpinel Because those fields will be present pre-merge and they don't matter for the functionality. Not sure if that helps.

Reply:
Right. So if it doesn't matter, why mention them? Is it because you can't just remove `md5` without removing the other fields due to the kind of YAML structure?

Reply:
Because people might wonder if those need to be preserved, merged or can be removed. All options are fine from dvc standpoint.

Reply:
I see. OK I think there's no need to mention all the fields. It's a bit confusing and not the goal here. I simplified it in eecda33e. Thanks!